data-prepper-api/src/test/java/org/opensearch/dataprepper/model/configuration/PipelinesDataFlowModelTest.java

```diff
@@ -60,6 +64,28 @@ void testSerializing_PipelinesDataFlowModel_empty_Plugins_with_nonEmpty_delay_an
         assertThat(serializedString, equalTo(expectedYaml));
     }
 
+    @Test
+    void testSerializing_PipelinesDataFlowModel_with_Version() throws JsonProcessingException {
```

Comment:
Thank you for adding new test cases (rather than modifying existing YAML files).